src/sentry/models/projectownership.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import absolute_import
+
+from jsonfield import JSONField
+
+from django.db import models
+from django.utils import timezone
+from sentry.db.models import Model, sane_repr
+from sentry.db.models.fields import FlexibleForeignKey
+
+
+class ProjectOwnership(Model):
+    __core__ = True
+
+    project = FlexibleForeignKey('sentry.Project', unique=True)
+    raw = models.TextField(null=True)
+    schema = JSONField(null=True)
```

Comment:
This will store the parsed/structured version. The intention of having two files is because of this is a plain text input, whitespace and formatting will be useful for readability. My grammar for the raw input also allows comments.

Reply:
I guess for more context, the result of this: https://gist.github.com/mattrobenolt/22faebaa6951f6d8f3bd7f480d045298